src/+otp/ProblemSettings.m

```diff
@@ -0,0 +1,18 @@
+classdef (Hidden) ProblemSettings
+    properties
+        ExpectedNumVars(1, 1) double {mustBeNonnegative}
```

Comment:
The spacing here is inconsistent with what you do in the brusselator one. Pick one IMO.


Also: why "double"?